data/map systems.txt

```diff
@@ -16924,6 +17922,65 @@ system Ik'kara'ka
 			distance 369
 			period 24.9998
 
+system Iklil
+	pos -291.11111 98.333335
+	government Syndicate
+	link Electra
+	link Tabit
+	asteroids "small rock" 9 1.1271
+	asteroids "medium rock" 1 1.4365
+	asteroids "large rock" 24 2.1216
+	asteroids "medium metal" 74 1.2376
+	asteroids "large metal" 1 1.3039
+	minables silicon 19 1.5395955
+	minables platinum 32 1.4626157
+	fleet "Large Core Pirates" 800
+	fleet "Small Core Pirates" 1200
+	fleet "Small Core Merchants" 600
```

Comment:
```suggestion
	fleet "Small Core Pirates" 1000
	fleet "Large Core Pirates" 2500
	fleet "Small Core Merchants" 2000
```
Just the pirates taking the back route and the occasional lost merchant. Pirate frequency here is the same as in Puppis.